IPython/html/static/notebook/js/kernelselector.js

```diff
@@ -78,7 +78,7 @@ define([
             new_notebook_submenu.append(
                 $("<li>").attr("id", "new-notebook-submenu-"+ks.name).append(
                     $('<a>')
-                        .attr('href', '#')
+                        .attr('href', 'javascript:void(0)')
```

Comment:
It however is an extremely common pattern, and I don't think it needs to or should change in this PR.